docs/UserGuide.md

```diff
@@ -27,7 +27,7 @@ DengueHotspotTracker Level 3 (AB3) is a **desktop app for managing contacts, opt
 
    * `list` : Lists all cases.
 
-   * `add n/John Tan p/543299 d/2023-02-13 19:52 a/20` : Adds a contact named `John Tan` to the Dengue Hotspot Tracker.
+   * `add n/John Tan p/543299 d/2023-02-13 a/20` : Adds a contact named `John Tan` to the Dengue Hotspot Tracker.
```

Comment:
All instances of "contact" in the User Guide should be changed to "case"!

Reply:
noted!